src/epggrab.c

```diff
@@ -328,8 +328,16 @@ epggrab_class_ota_cron_notify(void *self, const char *lang)
   epggrab_ota_set_cron();
 }
 
+static void
+epggrab_class_ota_genre_translation_notify(void *self, const char *lang)
+{
+  //tvhtrace(LS_SETTINGS, "Genre translation settings changed.");
```

Comment:
Remove any commented code if it's not necessary/wanted

Reply:
Done